src/mower_logic/src/monitoring/monitoring.cpp

```diff
@@ -178,9 +144,61 @@ void set_sensor_limits(SensorConfig &sensor_config) {
   if (sensor_config.si.upper_critical_value) sensor_config.si.has_critical_high = true;
 }
 
+void set_limits_battery_v(SensorConfig &sensor_config) {
+  sensor_config.si.lower_critical_value = mower_logic_config.battery_critical_voltage;
+  sensor_config.si.min_value = mower_logic_config.battery_empty_voltage;
+  sensor_config.si.max_value = mower_logic_config.battery_full_voltage;
+  sensor_config.si.upper_critical_value = mower_logic_config.battery_critical_high_voltage;
+  set_si_has_props(sensor_config);
+}
+
+void set_limits_charge_v(SensorConfig &sensor_config) {
+  // FIXME: Shall these better go to mower_logic's dyn-reconfigure (or an own dyn-reconfigure server)?
+  sensor_config.si.min_value = 24.0f;             // Mnimum voltage for before deep-discharge
+  sensor_config.si.max_value = 29.2f;             // Optimal charge voltage
+  sensor_config.si.upper_critical_value = 30.0f;  // Taken from OpenMower FW
+  set_si_has_props(sensor_config);
+}
+
+void set_limits_charge_current(SensorConfig &sensor_config) {
+  // FIXME: Shall these better go to mower_logic's dyn-reconfigure (or an own dyn-reconfigure server)?
+  sensor_config.si.max_value = 1.0f;             // Taken from the docs
+  sensor_config.si.upper_critical_value = 1.5f;  // Taken from OpenMower FW
+  set_si_has_props(sensor_config);
+}
+
+void set_limits_esc_temp(SensorConfig &sensor_config) {
+  sensor_config.si.max_value = paramNh->param(sensor_config.param_path + "/max_pcb_temp", 0);
+  set_si_has_props(sensor_config);
+}
+
+void set_limits_mow_motor_current(SensorConfig &sensor_config) {
+  sensor_config.si.upper_critical_value = paramNh->param(sensor_config.param_path + "/motor_current_limit", 0.0f);
+  set_si_has_props(sensor_config);
+}
+
+void set_limits_mow_motor_rpm(SensorConfig &sensor_config) {
+  // Use YF-C500 mow motor for default values
+  sensor_config.si.lower_critical_value = paramNh->param(sensor_config.param_path + "/min_motor_rpm_critical", 2300);
+  sensor_config.si.min_value = paramNh->param(sensor_config.param_path + "/min_motor_rpm", 2800);
+  sensor_config.si.max_value = paramNh->param(sensor_config.param_path + "/max_motor_rpm", 3800);
+  set_si_has_props(sensor_config);
+}
+
+void set_limits_mow_motor_temp(SensorConfig &sensor_config) {
+  // mower_config settings have precedence before xesc param file because user editable
+  sensor_config.si.max_value =
+      (mower_logic_config.motor_hot_temperature ? mower_logic_config.motor_hot_temperature
+                                                : paramNh->param(sensor_config.param_path + "/max_motor_temp", 0.0f));
```

Comment:
Do you know the [Elvis operator](https://en.wikipedia.org/wiki/Elvis_operator)? 😉 
```c++
sensor_config.si.max_value = mower_logic_config.motor_hot_temperature ?: paramNh->param(sensor_config.param_path + "/max_motor_temp", 0.0f));
```